src/tests/suite/daos_container.c

```diff
@@ -2072,7 +2072,8 @@ co_rf_simple(void **state)
 	test_arg_t		*arg0 = *state;
 	test_arg_t		*arg = NULL;
 	daos_obj_id_t		 oid;
-	daos_handle_t		 coh, oh;
+	daos_handle_t		 coh, oh, slave_coh;
```

Comment:
intel PR might be on us using such phrases ;-)
please use coh2 or temp_coh

Reply:
OK.